apps/api/.env.sandbox

```diff
@@ -0,0 +1,75 @@
+# Server Configuration
```

Comment:
This was the simplest approach I found to run both APIs at the same time. Just running `pnpm dev` and `NODE_ENV=sandbox pnpm dev` does the trick. Happy to update this if there’s a better way